FlowCrypt/Core/CoreTypes.swift

```diff
@@ -48,18 +48,26 @@ struct CoreRes {
         let key: KeyDetails
     }
     
-    struct DecryptFile: Decodable {
-        let name: String
-        let content: Data
+    struct DecryptFileWithoutData: Decodable {
+        let decryptSuccess: DecryptSuccess?
+        let decryptErr: DecryptErr?
+        struct DecryptSuccess: Decodable {
+            let name: String
+        }
+    }
+        
+    struct DecryptFile {
+        let decryptSuccess: DecryptSuccess?
+        let decryptErr: DecryptErr?
+        struct DecryptSuccess {
+            let name: String
+            let data: Data
+        }
```

Comment:
What about using only `DecryptFile` with optional `data` property instead of `DecryptFileWithoutData`? And then we can just check if `data == nil`

Reply:
We can even make it local to `decryptFile`, as it's not used anywhere else:
```swift
public func decryptFile(encrypted: Data, keys: [PrvKeyInfo], msgPwd: String?) async throws -> CoreRes.DecryptFile {
    struct DecryptFileRaw: Decodable {
        let decryptSuccess: DecryptSuccess?
        let decryptErr: DecryptErr?

        struct DecryptSuccess: Decodable {
            let name: String
        }
    }

    let json: [String : Any?]? = [
    ...
}
```